Adapted from libjitsi/resources/install/debian/patches/remove-gsm

Description: Removes gsm codec due to patent issues.
--- libjitsi/src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java.gsm
+++ libjitsi/src/org/jitsi/impl/neomedia/codec/FMJPlugInConfiguration.java
@@ -77,10 +77,10 @@
                 : null,
             "org.jitsi.impl.neomedia.codec.audio.g722.JNIDecoderImpl",
             "org.jitsi.impl.neomedia.codec.audio.g722.JNIEncoderImpl",
-            "org.jitsi.impl.neomedia.codec.audio.gsm.Decoder",
-            "org.jitsi.impl.neomedia.codec.audio.gsm.Encoder",
-            "org.jitsi.impl.neomedia.codec.audio.gsm.DePacketizer",
-            "org.jitsi.impl.neomedia.codec.audio.gsm.Packetizer",
+            //"org.jitsi.impl.neomedia.codec.audio.gsm.Decoder",
+            //"org.jitsi.impl.neomedia.codec.audio.gsm.Encoder",
+            //"org.jitsi.impl.neomedia.codec.audio.gsm.DePacketizer",
+            //"org.jitsi.impl.neomedia.codec.audio.gsm.Packetizer",
             "org.jitsi.impl.neomedia.codec.audio.silk.JavaDecoder",
             "org.jitsi.impl.neomedia.codec.audio.silk.JavaEncoder",
             //"org.jitsi.impl.neomedia.codec.video.h263p.DePacketizer",
